Makefile

```diff
@@ -73,7 +73,7 @@ prepare-release: cross
 
 .PHONY: test
 test:
-	go test -race $(PKGS)
+	go test -gcflags="-N -l" -race $(PKGS)
```

Comment:
can you please add comment about this new flag ? what's the purpose of it ?

Reply:
Ack